internal/linker/linker.go

```diff
@@ -64,23 +67,16 @@ func loadLinkerPatches() (string, map[string]string, error) {
 			if file.IsNew || file.IsDelete || file.IsCopy || file.IsRename {
 				panic("only modification patch is supported")
 			}
-			patches[file.OldName] = string(patchBytes)
+			modFiles = append(modFiles, file.OldName)
 		}
+		patches = append(patches, string(patchBytes))
```

Comment:
nit: you don't need the patches to be strings as far as I can tell, this could be `[][]byte` and you can replace `strings` with `bytes`